js/scrollsmooth.js

```diff
@@ -0,0 +1,18 @@
+const scroll_button=document.getElementsByClassName(".smooth-scroll");
+for(let i of scroll_button)
+{
+    i.addEventListener('click',function(e){
+        e.preventDefault();
+        const target=this.getAttribute('href');
```

Comment:
For writing a clean code, it is better to add space before and after equal mark.
For exemple: 
``` diff
- const target=this.getAttribute('href');
+ const target = this.getAttribute('href');
```

Reply:
thanks brother I won't do it again